File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        
        noDuplicate = []

        for x in nums:
            if x in noDuplicate:
                continue
            noDuplicate.append(x)
        
        numInstances = []

        for x in noDuplicate:
            numInstances.append(nums.count(x))

        output = []

        for x in range(k):
            output.append(noDuplicate[numInstances.index(max(numInstances))])
            numInstances[numInstances.index(max(numInstances))] = 0
        
        return output
        

File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        vector<int> output(2);
        for (int i = 0; i < numbers.size(); i++) {
            for (int j = i+1; j < numbers.size(); j++) {
                if (numbers[i] + numbers[j] == target) {
                    output[0] = i+1;
                    output[1] = j+1; 
                    return output;                   
                }
            }
        }
        
    }
};
